docs/toc.md

```diff
@@ -141,6 +141,7 @@
 ## [IncrementalLoadingCollection](helpers/IncrementalLoadingCollection.md)
 ## [NetworkHelper](helpers/NetworkHelper.md)
 ## [ObjectStorage](helpers/ObjectStorage.md)
+## [ParallelHelper](helpers/ParallelHelper.md)

```

Comment:
I know it's a `Helper` but do we want this under Developer Tools as that's where we put the Guard APIs for now until we tackle #303?

Reply:
Moved to another section as mentioned above with https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/301/commits/9ec494793dd7a02298803c70e50c41678be5508e and later commits 😊